src/FormParseNode.cs

```diff
@@ -208,7 +231,14 @@ public IEnumerable<T> GetCollectionOfPrimitiveValues<T>()
     {
         if(string.IsNullOrEmpty(rawValue))
             return null;
-        if(typeof(T).GetCustomAttributes<FlagsAttribute>().Any())
+
+        bool hasFlagsAttribute = false;
+        using (var enumerator = typeof(T).GetCustomAttributes<FlagsAttribute>().GetEnumerator())
+        {
+            hasFlagsAttribute = enumerator.MoveNext();
+        }
+
+        if (hasFlagsAttribute)
```

Comment:
This avoids creating a garbage instance of the attribute and enumerator as well (same comment to the other occurrence of this pattern in this PR):

```suggestion
        if (typeof(T).IsDefined(typeof(FlagsAttribute)))
```